### DISCUSSION TITLE
Paperless NGX dependency update MissingDependencyError: Ghostscript 10.0.0 through 10.02.0

### DISCUSSION THREAD

**Discussion by Stephan-4711** [tteck/Proxmox#4002] [Help Needed] [answered]
Hi,

the newest paperless ngx update seems to need a newer Ghostscript version.
Is that something you can fix in your script`?

**Comment by jsapede** [comment]
same error here  

`10-2024_bulletin_de_paie.pdf: Error occurred while consuming document 10-2024_bulletin_de_paie.pdf: MissingDependencyError: Ghostscript 10.0.0 through 10.02.0 (your version: 10.0.0) contain serious regressions that corrupt PDFs with existing text, such as those processed using --skip-text or --redo-ocr. Please upgrade to a newer version, or use --output-type pdf to avoid Ghostscript, or use --force-ocr to discard existing text.`

**Comment by barcelona92** [comment]
Same problem

**Reply by jsapede** [reply]
looks like its a ghostscript update to do, error is reported on paperless github

**Reply by Stephan-4711** [reply]
as I wrote?!

**Comment by MickLesk** [comment, marked as answer]
cd /tmp
wget https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10040/ghostscript-10.04.0.tar.gz
tar -xzf ghostscript-10.04.0.tar.gz
cd ghostscript-10.04.0
./configure
make
sudo make install

gs -v (Output looks like): 
GPL Ghostscript 10.04.0 
Copyright (C) 2024 Artifex Software, Inc.  All rights reserved.


i will fix it in the script asap. i use the 10.04 since a few weeks with paperless

**Reply by Stephan-4711** [reply]
Thx, it works. yea a general fix would be great. Is there a way to auto update ghostscript or is a manual compiling needed everytime? And to a need something like make clean afterwards? Long time ago that I complied a kernel 🙈

**Reply by MickLesk** [reply]
Actually Manual deploy needed, ITS Not in the Debian repo yet
